README.md

```diff
@@ -1,2 +1,45 @@
 # github-actions
 A collection of reusable and composite Github Actions
+
+## maven/build-scan-save
```

Comment:
I'm OK with this naming for now, but I suspect we might want to reorganize things in the long run.
(In fact, we should consider if we should try to combine the `setup-gradle` and `setup-maven` actions into the same repository as these actions.)

Reply:
Agreed